[2022-09-15T08:50:29.426+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: vecap_etl.date_dim manual__2022-09-15T08:50:26.825619+00:00 [queued]>
[2022-09-15T08:50:29.441+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: vecap_etl.date_dim manual__2022-09-15T08:50:26.825619+00:00 [queued]>
[2022-09-15T08:50:29.442+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-15T08:50:29.444+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-15T08:50:29.445+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-15T08:50:29.469+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): date_dim> on 2022-09-15 08:50:26.825619+00:00
[2022-09-15T08:50:29.478+0000] {standard_task_runner.py:52} INFO - Started process 19010 to run task
[2022-09-15T08:50:29.483+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vecap_etl', 'date_dim', 'manual__2022-09-15T08:50:26.825619+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/c_dag.py', '--cfg-path', '/tmp/tmpazqmznl3', '--error-file', '/tmp/tmp_cj1170w']
[2022-09-15T08:50:29.488+0000] {standard_task_runner.py:80} INFO - Job 94: Subtask date_dim
[2022-09-15T08:50:29.613+0000] {task_command.py:371} INFO - Running <TaskInstance: vecap_etl.date_dim manual__2022-09-15T08:50:26.825619+00:00 [running]> on host 0d275337f52b
[2022-09-15T08:50:29.763+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=admin@vecap.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vecap_etl
AIRFLOW_CTX_TASK_ID=date_dim
AIRFLOW_CTX_EXECUTION_DATE=2022-09-15T08:50:26.825619+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-15T08:50:26.825619+00:00
[2022-09-15T08:50:36.217+0000] {python.py:173} INFO - Done. Returned value was: <sqlalchemy.engine.cursor.LegacyCursorResult object at 0x7f73da2ddbd0>
[2022-09-15T08:50:36.241+0000] {xcom.py:586} ERROR - Could not serialize the XCom value into JSON. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config.
[2022-09-15T08:50:36.243+0000] {taskinstance.py:1902} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 68, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2405, in xcom_push
    session=session,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 68, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 198, in set
    map_index=map_index,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 583, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/local/lib/python3.7/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/local/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type LegacyCursorResult is not JSON serializable
[2022-09-15T08:50:36.267+0000] {taskinstance.py:1412} INFO - Marking task as FAILED. dag_id=vecap_etl, task_id=date_dim, execution_date=20220915T085026, start_date=20220915T085029, end_date=20220915T085036
[2022-09-15T08:50:36.291+0000] {standard_task_runner.py:97} ERROR - Failed to execute job 94 for task date_dim (Object of type LegacyCursorResult is not JSON serializable; 19010)
[2022-09-15T08:50:36.326+0000] {local_task_job.py:156} INFO - Task exited with return code 1
